core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -241,10 +275,10 @@ object StorageTool extends Logging {
   private def configToSelfManagedMode(config: KafkaConfig): Boolean = config.processRoles.nonEmpty
 
   def getMetadataVersion(
-    namespace: Namespace,
-    featureNamesAndLevelsMap: Map[String, java.lang.Short],
-    defaultVersionString: Option[String]
-  ): MetadataVersion = {
+                          namespace: Namespace,
+                          featureNamesAndLevelsMap: Map[String, java.lang.Short],
+                          defaultVersionString: Option[String]
+                        ): MetadataVersion = {
```

Comment:
Please preserve the old indentation. In Kafka that is how we indent parameters to a method that span multiple lines.